docs/wiki/CL/overview.md

```diff
@@ -1,20 +1,415 @@
+
+Ethereum's consensus protocol actually "bolts together" two different consensus protocols. One is called [LMD GHOST](/wiki/CL/cl-architecture), the other [Casper FFG](link to arch casper). The combination has become known as [Gasper](). In subsequent sections we will be looking at these both separately and in combination.
+
+## Proof-of-Work and Proof-of-stake
+
+This is a good point to clarify that neither Proof-of-Work (PoW) nor Proof-of-Stake (PoS) are consensus protocols by themselves. They are often incorrectly referred to as such, but they are actually mechanisms that enable consensus protocols. Both PoW and PoS primarily serve as Sybil resistance mechanisms, placing a cost on participating in the protocol. This prevents attackers from overwhelming the system cheaply.
+
+However, both PoW and PoS are closely linked to the consensus mechanisms they support through [fork choice](/wiki/link-to-fork-choice-cl-arch) rules. They help assign a weight or score to a chain of blocks: in PoW, it's the total computational work done; in PoS, it's the total value staked that supports a particular chain. Beyond these basics, PoW and PoS can support various consensus protocols, each with its own dynamics and trade-offs.
+
+### Block chains
+
+The basic element of a blockchain is the block. A block contains a set of transactions assembled by a leader (block proposer). The contents of a block can vary based on the protocol.
+
+- The payload of a block on Ethereum's execution chain is a list of user transactions.
+- In the pre-Merge PoS Beacon Chain, a block's payload was mostly a set of attestations by validators.
+- Post-Merge Beacon Chain blocks also include the execution payload (user transactions).
+- After [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844)(Deneb upgrade), blocks now also contain commitments to opaque data blobs alongside user transactions.
+
+Except for the Genesis block, each block builds on and points to a parent block, forming a chain of blocks: a blockchain. The goal is for all nodes on the network to agree on the same blockchain history.
+
+<a id="img_blockchain"></a>
+
+<figure class="diagram" style="text-align:center">
+
+![Diagram for Block Chain](../../images/cl/blockchain.svg)
+
+<figcaption>
+
+_Time moves from left to right and, except for the Genesis block, each block points to the parent block it builds on._
+
+</figcaption>
+</figure>
+
+The chain grows as nodes add new blocks to its tip. This is done by temporarily selecting a "leader", the node that extends the chain. In PoW, the leader is the miner who first solves the PoW puzzle for its block. In Ethereum's PoS, the leader is pseudo-randomly selected from active stakers.
+
```

Comment:
I would ditch bitcoin and blocksize war reference here. Just mention the gas limit of block which limits its size and enables set a baseline for node resources

Reply:
removed it